zebra/zebra_evpn_mac.c

```diff
@@ -1006,6 +1006,15 @@ int zebra_evpn_mac_del(zebra_evpn_t *zevpn, zebra_mac_t *mac)
 			   mac->flags);
 	}
 
+	/* If the MAC is freed before the neigh we will end up
+	 * with a stale pointer against the neigh
+	 */
+	if (!list_isempty(mac->neigh_list))
+		zlog_warn("%s: MAC %s flags 0x%x neigh list not empty %d",
```

Comment:
I suggest starting using `%pEA` for `prefix_mac2str` everywhere where possible.

Reply:
Sure, will fixup